docs/docs/upgrade-gatsby-and-dependencies.md

```diff
@@ -0,0 +1,142 @@
+---
+title: Upgrade Gatsby for minor or patch releases
+---
+
+## Introduction
+
+To keep up with the latest bug fixes, security patches and minor releases from both Gatsby and its dependencies, you should constantly upgrade to the latest version of each one.
+
+In this guide you will learn how to upgrade Gatsby and its dependencies for minor or patch releases.
+
+## Semantic versioning
+
+As many other pacakages, Gatsby uses [semantic versioning](https://semver.org/) to tag new versions and indicate what kind of changes are introduced in every new release.
+
+This guide is meant to teach you how to upgrade Gatsby for minor or patch releases. For major changes you can refer to our [Release and Migrations](https://www.gatsbyjs.org/docs/releases-and-migration/) reference guide overview for the corresponding guide to upgrade.
+
+## Why you should upgrade Gatsby and its dependencies
+
+Every new version of every package comes with improvements on multiple categories from performance, accesibility, to security and bug fixes, among others, so it is important to upgrade both Gatsby and its dependencies to get the latest improvements in every one of these categories.
+
+Constantly upgrading your dependencies on minor or patch releases also helps you to make major upgrades easier and to identify ossible soon to be deprecated functionality or APIs.
+
+## How to identify possible upgrades
+
+Depending on your current setup, you can use one of [yarn](https://yarnpkg.com/) or [npm](https://www.npmjs.com/get-npm) to update your dependencies.
```

Comment:
We recommend npm throughout the docs to keep things simple (with one exception being in the contributing docs for the Gatsby repo), so we should be consistent with that here and remove the Yarn instructions.